### PR TITLE
Miscellaneous bug fixes.

### PR DESCRIPTION
**Pull request by armcburney**
@davepagurek

### CODE REVIEW

**Review by davepagurek — APPROVED** [review]
this is very appealing 🍌

File: lib/emerald/nodes/each.rb [hunk 1]
```diff
@@ -31,6 +31,8 @@ def to_html(body, context)
           body.to_html(new_ctx)
         end
         .join("\n")
+    elsif vars.nil?
+      next
```
Comment (davepagurek) [inline lib/emerald/nodes/each.rb]: Actually wait need to change this, just `''` empty string here

**Review by davepagurek — CHANGES_REQUESTED** [review]

File: lib/emerald/nodes/each.rb (hunk 1 above)
Comment (davepagurek) [reply lib/emerald/nodes/each.rb]: Because we aren't in a block, this is just the return value

**Comment by davepagurek** [comment]
I peeled too soon, rip :( 🍐

File: lib/emerald/nodes/each.rb (hunk 1 above)
Comment (armcburney) [reply lib/emerald/nodes/each.rb]: Oh shoot, oh no :'(

**Comment by armcburney** [comment]
Rip, oops

**Review by davepagurek — APPROVED** [review]

**Pull request merged**